python/paddle/hapi/model.py

```diff
@@ -649,19 +681,30 @@ def train_batch(self, inputs, labels=None):
         labels = labels or []
         labels = [to_variable(l) for l in to_list(labels)]
 
-        if self._nranks > 1:
-            outputs = self.ddp_model.forward(* [to_variable(x) for x in inputs])
-        else:
-            outputs = self.model.network.forward(
-                * [to_variable(x) for x in inputs])
+        scaler = paddle.amp.GradScaler(**self._amp_configs)
```

Comment:
Are all key-values in self._amp_configs accepted by GradScaler?

Reply:
Thanks! Done. I added input check for `amp_configs`. Thanks:)